pom.xml

```diff
@@ -528,10 +534,53 @@
           <configuration>
             <analysisConfiguration>
               <versionFormat>[-0-9.]*</versionFormat>
-              <failBuildOnProblemsFound>true</failBuildOnProblemsFound>
-              <revapi.semver.ignore>
+              <failBuildOnProblemsFound>false</failBuildOnProblemsFound>
+              <revapi.versions>
                 <enabled>true</enabled>
-              </revapi.semver.ignore>
+              </revapi.versions>
+              <!-- Exclude of the hudson package not working-->
```

Comment:
Maybe it would make sense to report that problem as RevAPI issue?

Reply:
yes definitely. This problem can happen to others as well. I will create an issue.